docs/js/templateData.js

```diff
@@ -19,6 +19,6 @@ limitations under the License.
 var context={
     "version": "23",
     "dotVersion": "2.3",
-    "fullDotVersion": "2.3.1",
+    "fullDotVersion": "2.3.3",
```

Comment:
Shouldn't this be 2.3.2? The latest kafka release is 2.3.1.

Reply:
Its interesting, because elsewhere the version was already at 2.3.2, here it was skipped being bumped to 2.3.2 - Hence the change now is to 2.3.2